codefest-client/src/main/kotlin/codefest/client/TopController.kt

```diff
@@ -15,6 +16,9 @@ class TopController {
     @FXML
     private lateinit var listView: ListView<Student>
 
+    @FXML
+    private lateinit var tv: Text
```

Comment:
Can we rename tv to something more meaningful?

Reply:
Yes, "tv" could be renamed to "leaderboardHeader", however, this was scaffolding as I wasn't sure if I could create a well-formatted header via javaFX TableView. 

Since this pull request's main improvement was formatting Students, an alternative is to revert the "tv" commit until a better solution can be provided, what's your opinion?